apps/meeting/src/providers/AppStateProvider.tsx

```diff
@@ -26,6 +27,7 @@ interface AppStateValue {
   keepLastFrameWhenPaused: boolean;
   layout: Layout;
   joinInfo: JoinMeetingInfo | undefined;
+  backgroundReplacementOption: string;
```

Comment:
better to make it enum or string literal instead of string.

Reply:
may I know what id the benefit of that? The above params are using string as type.